feature/external-credential/src/main/java/com/simprints/feature/externalcredential/screens/scanocr/ExternalCredentialScanOcrViewModel.kt

```diff
@@ -67,10 +66,10 @@ internal class ExternalCredentialScanOcrViewModel @AssistedInject constructor(
     private var state: ScanOcrState = ScanOcrState.EMPTY
         set(value) {
             field = value
-            _stateLiveData.postValue(value)
+            _scanOcrStateLiveData.postValue(value)
         }
-    private val _stateLiveData = MutableLiveData<ScanOcrState>()
-    val stateLiveData: LiveData<ScanOcrState> = _stateLiveData
+    private val _scanOcrStateLiveData = MutableLiveData(state)
```

Comment:
Do we need both `state` and `_scanOcrStateLiveData`?

Reply:
My point was can't we set the initial and all following states to `_scanOcrStateLiveData` directly and do without a second private var?